frontend/integration-tests/tests/add-secrets.scenario.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable max-nested-callbacks */
+import { appHost, testName, checkLogs, checkErrors } from '../protractor.conf';
+import { execSync } from 'child_process';
+import * as secretsView from '../views/secrets.view';
+import * as addSecretView from '../views/modal-add-secret-to-workload.view';
+import { searchJSON } from '../views/modal-add-secret-to-workload.view';
+
+describe('Add Secret to Workloads', () => {
+  const secretName = 'test-secret';
+  const resourceName = 'example';
+  const resourceKind = 'deployment';
+  const envPrefix = 'env-';
+  const mountPath = '/tmp/testdata';
+
+
+  beforeAll(async()=> {
+    // create deployment and secret
+    execSync(`kubectl run ${resourceName} --image=aosqe/hello-openshift -n ${testName}`);
```

Comment:
Not sure if we shouldn't create a new secret and deployment for each TC. @spadgett thoughts ?

Reply:
I think it's OK. I'm worried we might get name conflicts if we create and delete the obj before and after each tests since delete doesn't happen immediately.

Reply:
How about changing resource name to be unique? Such as `${case_id}-type`